components/NavLayout/SideBar.js

```diff
@@ -1,12 +1,183 @@
+  justifyContent: 'flex-end',
+  padding: theme.spacing(0, 1),
+  // necessary for content to be below app bar
+  ...theme.mixins.toolbar,
+}));
+
+const AppBar = styled(MuiAppBar, {
+  shouldForwardProp: (prop) => prop !== 'open',
+})(({ theme, open }) => ({
+  zIndex: theme.zIndex.drawer + 1,
+  transition: theme.transitions.create(['width', 'margin'], {
+    easing: theme.transitions.easing.sharp,
+    duration: theme.transitions.duration.leavingScreen,
+  }),
+  ...(open && {
+    marginLeft: drawerWidth,
+    width: `calc(100% - ${drawerWidth}px)`,
+    transition: theme.transitions.create(['width', 'margin'], {
+      easing: theme.transitions.easing.sharp,
+      duration: theme.transitions.duration.enteringScreen,
+    }),
+  }),
+}));
+
+const Drawer = styled(MuiDrawer, { shouldForwardProp: (prop) => prop !== 'open' })(
+  ({ theme, open }) => ({
+    width: drawerWidth,
+    flexShrink: 0,
+    whiteSpace: 'nowrap',
+    boxSizing: 'border-box',
+    ...(open && {
+      ...openedMixin(theme),
+      '& .MuiDrawer-paper': openedMixin(theme),
+    }),
+    ...(!open && {
+      ...closedMixin(theme),
+      '& .MuiDrawer-paper': closedMixin(theme),
+    }),
+  }),
+);
 
 export default function SideBar({ children, ...props }) {
+  const theme = useTheme();
+  const [open, setOpen] = useState(false);
+
+  const handleDrawerOpen = () => {
+    setOpen(true);
+  };
+
+  const handleDrawerClose = () => {
+    setOpen(false);
+  };
+
+  // hard coded data, eventually change sampleUser to [username]
+  const pageUrls = {
+    0: "/sampleUser/my-classes",
+    1: "/sampleUser/calendar",
+    2: "/sampleUser/notifications",
+    3: "/sampleUser/class-catalog"
+  }
+
+  // TODO: change "Current User" to [username]'s name
+  // TODO: change avatar src to dynamically render user's profile picture
   return (
-    <nav className={styles.nav}>
-      <h3>My Classes</h3>
-      <h3>Calendar</h3>
-      <h3>Notifications</h3>
-      <h3>Class Catalog</h3>
-    </nav>
+    <Box sx={{ display: 'flex' }}>
+      <CssBaseline />
+      <AppBar position="fixed" open={open} className={styles.topbar}>
+        <Toolbar>
+          <IconButton
+            color="inherit"
+            aria-label="open drawer"
+            onClick={handleDrawerOpen}
+            edge="start"
+            sx={{
+              marginRight: 5,
+              ...(open && { display: 'none' }),
+            }}
+          >
+            <MenuIcon />
+          </IconButton>
+          <Typography className={styles.siteTitle} variant="h5" noWrap component="div">
+            ClassMates
+          </Typography>
+          <Container className={styles.profileIcon} sx={{ flexGrow: 0 }}>
+            <Typography className={styles.profileName}>Current User</Typography>
+            <Avatar
+              className={styles.profileAvatar}
+              alt="Remy Sharp"
+              src="https://www.vhv.rs/dpng/d/426-4264903_user-avatar-png-picture-avatar-profile-dummy-transparent.png"
```

Comment:
Move image url to constants folder

Reply:
Thanks for the point! I will hop on this now

Reply:
I forgot about our constants folder, will move it right now!